waspc/src/Wasp/Generator/Node/Version.hs

```diff
@@ -49,11 +81,21 @@ parseNodeVersion nodeVersionStr =
       return $ SV.Version mjr mnr ptc
     _ -> Nothing
 
+nodeNotFoundMessage :: String
+nodeNotFoundMessage = "`node` command not found!"
+
+nodeUnknownErrorMessage :: IOError -> String
+nodeUnknownErrorMessage err =
```

Comment:
Maybe `makeUnknownNodeErrorMessage` since we're dealing with a function? I haven't programmed in Haskell for a while and forgot most of the conventions, so correct me if I'm saying nonsense.

Reply:
That makes sense, it will make it consistent with `makeNodeVersionMismatchMessage` at the very least.